Source/Libraries/CorruptCore/StockpileManager_UISide.cs

```diff
@@ -304,13 +304,13 @@ public static bool MergeStashkeys(List<StashKey> sks, bool _loadBeforeOperation
             return false;
         }
 
-        public static bool LoadState(StashKey sk, bool reloadRom = true, bool applyBlastLayer = true)
```

Comment:
Can you undo this removal and fix the line below (309) to make the second param `reloadRom` instead of always being true? Should probably double-check usages of the method as well.

Reply:
Done